src/express/responses/index.js

```diff
@@ -56,24 +56,29 @@ const errorResponse = (req, res, httpError, errorsList) => {
 
 module.exports = {
   successResponse: (req, res, result) => {
-    if (res.hasFinished) {
-      log.warn('[swagger response] The response to the current request has already been sent')
+    if (req.timedout) {
+      log.error('Request timeout, response is not sent')
       return
     }
 
     res.status(200).json(result)
   },
 
   createdResponse: (req, res, result) => {
-    if (res.hasFinished) {
-      log.warn('[swagger response] The response to the current request has already been sent')
+    if (req.timedout) {
+      log.error('Request timeout, response is not sent')
       return
     }
 
     res.status(201).json(result)
   },
 
   noContentResponse: (req, res) => {
+    if (req.timedout) {
+      log.error('Request timeout, response is not sent')
```

Comment:
Let's update copy to use `timedout` — I was not sure if it correct to use it, but they use it in `req.timedout`. Then please merge and update `dyno` service as well.

Reply:
So should be `'Request timedout, response is not sent'`